src/tests/mod.rs

```diff
@@ -71,20 +72,21 @@ impl SolverEngine {
     }
 
     /// Solves a raw JSON auction.
-    pub async fn solve(&self, auction: serde_json::Value) -> serde_json::Value {
+    pub async fn solve(&self, auction: serde_json::Value) -> anyhow::Result<serde_json::Value> {
         let client = reqwest::Client::new();
         let url = shared::url::join(&self.url, "solve");
-        let response = client.post(url).json(&auction).send().await.unwrap();
+        let response = client.post(url).json(&auction).send().await?;
```

Comment:
Do we now have a test where we want this to fail so we can assert on the error?
Otherwise it's probably nicer to just continue using `.unwrap()` in this function instead of at every call site.

Reply:
I forgot to add a comment. When this function fails, the error is not visible in the output since it panics inside the async function. The workaround is to return a result and panic inside the test.